src/drivers/ammo-driver.js

```diff
@@ -85,12 +91,16 @@ AmmoDriver.prototype.updateBody = function(body) {
 AmmoDriver.prototype.step = function(deltaTime) {
   this.physicsWorld.stepSimulation(deltaTime, this.maxSubSteps, this.fixedTimeStep);
 
+  const collisions = this.objectPool.obtain();
+  const prevCollisions = this.prevCollisions;
+
   const numManifolds = this.dispatcher.getNumManifolds();
   for (let i = 0; i < numManifolds; i++) {
     const persistentManifold = this.dispatcher.getManifoldByIndexInternal(i);
     const numContacts = persistentManifold.getNumContacts();
     const body0ptr = Ammo.getPointer(persistentManifold.getBody0());
     const body1ptr = Ammo.getPointer(persistentManifold.getBody1());
+    const collisionKey = body0ptr + ':' + body1ptr;
```

Comment:
Earlier versions of this code created the "key" this way, but I ultimately scrapped it because the string concatenation generates gc garbage every frame.